cocotb/drivers/amba.py

```diff
@@ -236,10 +236,11 @@ class AXI4Slave(BusDriver):
         "BID",     "RID",     "WID"
     ]
 
-    def __init__(self, entity, name, clock, memory, callback=None, event=None,
-                 big_endian=False):
-
-        BusDriver.__init__(self, entity, name, clock)
+    def __init__(self, entity, name, clock, memory, **kwargs):
+        callback = kwargs.pop('callback', None)
+        event = kwargs.pop('event', None)
+        big_endian = kwargs.pop('big_endian', False)
```

Comment:
Making these keyword-only when they used not to be is a breaking change, sadly. I don't think we should risk breaking existing users for such minor cleanup.

Reply:
I'll move it to follow-up patch.